### ISSUE TITLE
Runner Offline

### ISSUE THREAD

**Issue by pvelesko** [dkurt/github_actions_riscv#9]
I was finally able to get this compiled and configured

```
user@starfive:~/extraced$ ./config.sh --url xxxx --token xxxx 
ldd: ./bin/libcoreclr.so: No such file or directory
ldd: ./bin/libSystem.Security.Cryptography.Native.OpenSsl.so: No such file or directory
ldd: ./bin/libSystem.IO.Compression.Native.so: No such file or directory

--------------------------------------------------------------------------------
|        ____ _ _   _   _       _          _        _   _                      |
|       / ___(_) |_| | | |_   _| |__      / \   ___| |_(_) ___  _ __  ___      |
|      | |  _| | __| |_| | | | | '_ \    / _ \ / __| __| |/ _ \| '_ \/ __|     |
|      | |_| | | |_|  _  | |_| | |_) |  / ___ \ (__| |_| | (_) | | | \__ \     |
|       \____|_|\__|_| |_|\__,_|_.__/  /_/   \_\___|\__|_|\___/|_| |_|___/     |
|                                                                              |
|                       Self-hosted runner registration                        |
|                                                                              |
--------------------------------------------------------------------------------

# Authentication


√ Connected to GitHub

# Runner Registration

Enter the name of the runner group to add this runner to: [press Enter for Default]

Enter the name of runner: [press Enter for starfive]

This runner will have the following labels: 'self-hosted', 'Linux', 'RISCV64'
Enter any additional labels (ex. label-1,label-2): [press Enter to skip]

√ Runner successfully added
√ Runner connection is good

# Runner settings

Enter name of work folder: [press Enter for _work]

√ Settings Saved.
```

but it's always offline :( 

<img width="819" alt="image" src="https://github.com/user-attachments/assets/c1c8f21f-441b-4d14-8f0f-d92425e71542">

Did you encounter this issue as well or have any debug tips? 

I ran config network test and it passed all tests

**Comment by pvelesko** [comment]
fixed by rebasing on top of latest release

**Comment by woqidaideshi** [comment]
@pvelesko @dkurt I also encountered this issue. Did you rebased  branch [v2.321.0](https://github.com/actions/runner/releases/tag/v2.321.0)  of this repository: <https://github.com/actions/runner>? Thanks!

**Comment by pvelesko** [comment]
@woqidaideshi  my repo needs to be rebased now again.

**Comment by dkurt** [comment]
just published 2.321.0, sorry for delay: https://github.com/dkurt/github_actions_riscv/releases/tag/v2.321.0
